core/db/migrate/20150314013438_add_missing_indexes_on_spree_tables.rb

```diff
@@ -0,0 +1,57 @@
+class AddMissingIndexesOnSpreeTables < ActiveRecord::Migration
+  def change
+    unless index_exists?(:spree_promotion_rules_users, [:user_id, :promotion_rule_id])
+      add_index   :spree_promotion_rules_users,
```

Comment:
@krzysiek1507 The [documentation](http://apidock.com/rails/ActiveRecord/ConnectionAdapters/SchemaStatements/index_exists%3F) says we need to use table name and column name(s) as arguments.

I'll fix the spacing between `add_index` and arguments.